src/NUnitEngine/nunit.engine.tests/Services/ResultWriters/NUnit2XmlResultWriterTests.cs

```diff
@@ -1,340 +0,0 @@
-﻿// ***********************************************************************
```

Comment:
These two files weren't referenced in the tests and should have been deleted when the result writer was moved out to an extension.